src/styles/index.css

```diff
@@ -0,0 +1,210 @@
+body {
+  margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
+}
+
+:root {
+  --color-blue: #00acdc;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 60px;
+  padding: 0 50px;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+  display: block;
+}
+
+.nav__list {
+  display: flex;
+  align-items: center;
+  margin: 0;
+  padding: 0;
+  list-style: none;
+}
+
+.link {
+  position: relative;
+  display: flex;
+  align-items: center;
+  margin-left: 20px;
+  height: 60px;
+  font-size: 12px;
+  font-weight: 500;
+  text-align: left;
+  text-transform: uppercase;
+  color: black;
+  text-decoration: none;
+}
+
+.link:hover {
+  color: var(--color-blue);
+}
+
+.nav__list li:first-child .link {
+  margin-left: 0;
+}
+
+.link--is_active {
+  color: var(--color-blue);
+}
+
+.link.is-active::after {
+  content: '';
+  position: absolute;
+  left: 50%;
+  transform: translateX(-50%);
+  bottom: 0;
+  width: 100%;
+  height: 4px;
+  border-radius: 8px;
+  background: var(--color-blue);
+}
```

Comment:
The class selector '.link.is-active::after' is inconsistent with '.link--is_active'. Consider using a consistent naming convention for active states to avoid confusion and potential styling issues.